### ISSUE TITLE
pytest - show markers used in tests

### ISSUE THREAD

**Issue by MarcusJellinghaus** [MarcusJellinghaus/mcp-code-checker#47]
There should be a way for the mcp client to get all markdowns used in tests

Techncical first evaluation below

````markdown
## Listing pytest markers

There are two kinds of markers to list:

### 1. Registered markers (declared in config or provided by plugins)
Run:
```bash
pytest --markers
````

This shows all markers pytest knows about, including built-ins and any custom markers defined in `pytest.ini`, `pyproject.toml`, or via plugins.

---

#### Option A: Use `--setup-plan`

```bash
pytest --setup-plan -q
```

This outputs each collected test with its markers.

#### Option B: Collect markers programmatically

Add the following to `conftest.py`:

```python
import pytest

def pytest_collection_finish(session):
    markers = set()
    for item in session.items:
        for mark in item.iter_markers():
            markers.add(mark.name)
    print("\nMarkers used in tests:", sorted(markers))
```

Then run:

```bash
pytest -q
```

At the end you’ll see the unique markers that were actually used.

---

### 3. Quick static scan

If you just want to search for markers in the codebase:

```bash
grep -R "pytest.mark." tests/
```

```

Do you want me to also include an example `pytest.ini` section with marker registration for completeness?
```
